packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -38,6 +38,14 @@ const SEO = ({ title, location, children }) => {
       ? location.pathname
       : location.pathname.replace(new RegExp(`^\\/${locale.locale}`), '/');
 
+  const getSwiftypeSiteType = () => {
+    const nrSubDomain = /.*\.newrelic\.com/.test(location.hostname)
+      ? location.hostname.split('.')[0]
+      : 'demo';
```

Comment:
I'm not a huge fan of this fallback. While this is ok for the demo site itself, this is a very strange value for any other site. Could we have a bit more sensible default?

Reply:
yeah i was actually waffling between making it `null` if not running on a NR subdomain, which would remove the `<meta>` tag if kept as is. which is maybe fine? OR we could just let use localhost, but that feels a little weird too.

Reply:
I'd opt for removing the meta tag. If we want to "auto detect" the type rather than make it configurable, I think it makes total sense that we'd opt for not having it, rather than being an incorrect value.